apps/docs/src/principles/contributing.md

```diff
@@ -14,8 +14,7 @@ It gives tips and guidelines on what should or must be considered when working w
 
 ## Prerequisites / Technical setup
 
-1. Install [Node.js](https://nodejs.org/en) version {{ nodeVersion }}.
-
+1. Install [Node.js](https://nodejs.org/en) version **{{ nodeVersion }}**.
```

Comment:
the double space is important so there will still be a line break:
```suggestion
1. Install [Node.js](https://nodejs.org/en) version **{{ nodeVersion }}**.  
```

Reply:
I needed to use `<br />` because trailing spaces are removed by prettier